scss/_patterns_strip.scss

```diff
@@ -81,7 +81,7 @@
     background-repeat: no-repeat;
     background-size: cover;
 
-    &.is-light {
+    &.is-white {
```

Comment:
why this change?

Reply:
Reserving the name 'light' for a theme based on f7f7f7 background, as in the p-strip--light.

Reply:
changed to light